contributors/devel/conformance-tests.md

```diff
@@ -1,110 +1,150 @@
+
+- node/platform-reliant features, eg: multiple disk mounts, GPUs, high density,
+  etc.
+- optional features, eg: policy enforcement
+- cloud-provider-specific features, eg: GCE monitoring, S3 Bucketing, etc.
+- anything that requires a non-default admission plugin
+
+Examples of tests which are not eligible for promotion to conformance:
+- anything that checks specific Events are generated, as we make no guarantees
+  about the contents of events, nor their delivery
+- anything that checks optional Condition fields, such as Reason or Message, as
+  these may change over time (however it is reasonable to verify these fields
+  exist or are non-empty)
+
+## Conformance Test Version Skew Policy
 
 As each new release of Kubernetes provides new functionality, the subset of
 tests necessary to demonstrate conformance grows with each release. Conformance
 is thus considered versioned, with the same backwards compatibility guarantees
-as laid out in [our versioning policy](/contributors/design-proposals/release/versioning.md#supported-releases-and-component-skew).
-Conformance tests for a given version should be run off of the release branch
-that corresponds to that version. Thus `v1.2` conformance tests would be run
-from the head of the `release-1.2` branch. eg:
+as laid out in the [kubernetes versioning policy]
+
+To quote:
 
- - A v1.3 development cluster should pass v1.1, v1.2 conformance tests
+> For example, a v1.3 master should work with v1.1, v1.2, and v1.3 nodes, and 
+> should work with v1.2, v1.3, and v1.4 clients.
 
- - A v1.2 cluster should pass v1.1, v1.2 conformance tests
+Conformance tests for a given version should be run off of the release branch
+that corresponds to that version. Thus `v1.2` conformance tests would be run
+from the head of the `release-1.2` branch.
 
- - A v1.1 cluster should pass v1.0, v1.1 conformance tests, and fail v1.2
-conformance tests
+For example, suppose we're in the midst of developing kubernetes v1.3. The
+following clusters must pass conformance tests built from the following 
+branches:
 
+| cluster version | master | release-1.3 | release-1.2 | release-1.1 |
+| --------------- | -----  | ----------- | ----------- | ----------- |
+| v1.3.0-alpha    | yes    | yes         | yes         | no          |
+| v1.2.x          | no     | no          | yes         | yes         |
+| v1.1.x          | no     | no          | no          | yes         |
 
-### Running Conformance Tests
+## Running Conformance Tests
 
 Conformance tests are designed to be run with no cloud provider configured.
-Conformance tests can be run against clusters that have not been created with
-`hack/e2e.go`, just provide a kubeconfig with the appropriate endpoint and
-credentials.
+Conformance tests must be able to be run against clusters that have not been
+created with `hack/e2e.go`, just provide a kubeconfig with the appropriate
+endpoint and credentials.
 
 ```sh
+# ensure kubetest is installed
```

Comment:
For completeness, do you want to say that they have to have the k8s src tree cloned first since not everyone running the tests could be a k8s dev?

Reply:
It is possible to do this without the k8s src tree, just using the released `.tar.gz` and `-test.tar.gz` artifacts

Reply:
can we mention that then?  ie. show the URL where they can download the tar-ball from?  I prefer to assume we need to be pretty explicit for people - just to be extra helpful.